buildSrc/src/main/groovy/org/elasticsearch/gradle/test/StandaloneTestPlugin.groovy

```diff
@@ -46,5 +46,10 @@ public class StandaloneTestPlugin implements Plugin<Project> {
         test.testClassesDir project.sourceSets.test.output.classesDir
         test.mustRunAfter(project.precommit)
         project.check.dependsOn(test)
+
+        project.namingConventionsCheck {
+            // Stand alone tests don't follow the rules!
+            skipIntegTestInDisguise = true
```

Comment:
Yes, we should make it clear which projects have violators so we can fix them individually, and not allow others to pop up.